tests/basic/keymap.c

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2017 Fred Sundvik
+/* Copyright 2020 Google LLC
```

Comment:
Do not remove copyright notices.

```suggestion
/* Copyright 2017 Fred Sundvik
```